src/ui/public/timepicker/kbn_global_timepicker.js

```diff
@@ -19,7 +19,8 @@ UiModules
   return {
     template: toggleHtml,
     replace: true,
-    link: ($scope) => {
+    require: '^kbnTopNav',
```

Comment:
Do we need to require this? Can we add `onFilterSelect` and `onIntervalSelect` callbacks, and follow the same pattern as `timerpicker.js`? This would be a little more explicit, and less brittle.